packages/core/src/Stores/ui-store.js

```diff
@@ -550,8 +550,9 @@ export default class UIStore extends BaseStore {
     }
 
     @action.bound
-    toggleAccountTransferLimitModal(state_change = !this.is_account_transfer_limit_modal_visible) {
+    toggleAccountTransferLimitModal({ state_change: !this.is_account_transfer_limit_modal_visible, message }) {
```

Comment:
My bad, I think my previous suggestion was incorrect. Try this one: 
```suggestion
    toggleAccountTransferLimitModal({ state_change = !this.is_account_transfer_limit_modal_visible, message }) {
```

Reply:
No problem, I fixed it in my latest commit